tests/cypress/config.config.js

```diff
@@ -0,0 +1,18 @@
+const { defineConfig } = require('cypress')
```

Comment:
Can we rename this to `config.js`

Reply:
We can. Then it should be changed in `package.json` file too. That's all, I believe. I didn't change as it was automatically migrated from `config.json` by cypress.